packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2378,8 +2337,8 @@ export class ContainerRuntime
 	) {
 		this.addMetadataToSummary(summaryTree);
 
-		if (this._idCompressor) {
-			const idCompressorState = JSON.stringify(this._idCompressor.serialize(false));
+		if (this.idCompressor) {
+			const idCompressorState = JSON.stringify(this.idCompressor.serialize(false));
```

Comment:
Note that in the "delayed" state, the IdCompressor has been created, but is excluded from the summary because we have not yet used it.